cmd/rule-evaluator/main.go

```diff
@@ -67,6 +67,16 @@ import (
 
 const projectIDVar = "PROJECT_ID"
 
+var googleCloudBaseURL = urlMustParse("https://console.cloud.google.com/monitoring/metrics-explorer")
+
+func urlMustParse(s string) url.URL {
+	u, err := url.Parse(s)
+	if err != nil {
+		panic(err)
```

Comment:
Suggestion: Instead of using this helper that can panic, why not just define the literal?

```
url.URL{
	Scheme: "https",
	Host:   "console.cloud.google.com",
	Path:   "/monitoring/metrics-explorer",
}
```

Reply:
Great idea. Thanks!